src/components/appointment/AppointmentCard.tsx

```diff
@@ -0,0 +1,59 @@
+import { Appointment } from "@/types/Appointment";
+import { Card, Avatar, CardContent, CardHeader, IconButton, Stack, Typography } from "@mui/material";
+import { PagePaths } from "enum/pages";
+import router from "next/router";
+
+import LocationOnIcon from "@mui/icons-material/LocationOn";
+import CalendarTodayIcon from "@mui/icons-material/CalendarToday";
+import PersonIcon from '@mui/icons-material/Person';
+
+type props = {
+    appointment: Appointment;
+};
+export default function AppointmentCard(props: props) {
+    return <Card sx={{ width: "100%" }}>
+        <CardHeader
+            avatar={
+                <IconButton
+                    onClick={() => {
+                        router.push(PagePaths.profile + props.appointment.ownerId);
+                    }}
+                    sx={{ padding: "0px" }}
+                >
+                    <Avatar
+                        sx={{ width: "50px", height: "50px" }}
+                        alt="Profile picture"
+                        src={props.appointment.ownerProfilePic}
+                    />
+                </IconButton>
+            }
+            title={props.appointment.title}
+            subheader={"by " + props.appointment.ownerName}
+            titleTypographyProps={{ variant: "h5" }}
```

Comment:
We don't have `h5 h6` in our design system, Please determine an appropriate font settings to _app.tsx. (I trust your decision!)

Reply:
Not to mention, your decision might be used by other pages related to this hehe:grin:. (I also trust your decision!)

Reply:
Well, I add font for h5,h6 and I also changed fontsize for h1. Hope my decision were right though lol